ctapipe/image/showermodel_predictor.py

```diff
@@ -0,0 +1,148 @@
+import numpy as np
+from erfa.ufunc import s2p as spherical_to_cartesian
+
+__all__ = ["ShowermodelPredictor"]
+
+
+class ShowermodelPredictor:
+    def __init__(
```

Comment:
This should just get a `SubarrayDescription` I think and construct the things from there